dbterd/adapters/targets/dbml/engine/engine.py

```diff
@@ -85,25 +85,13 @@ def get_tables(manifest):
     ]
     for table in tables:
         parser = Parser(table.raw_sql)
-        try:
-            column_names = parser.columns_aliases_names
-        except AttributeError:
-            # Object has no associated column names, continue to next object
-            pass
 
+        column_names = getattr(parser, "columns_aliases_names", None)
         if column_names:
             for column in column_names:
-                table.columns.append(
-                    Column(
-                        name=column,
-                    )
-                )
+                table.columns.append(Column(name=column))
```

Comment:
I'm using `black` auto-format here - not sure if it would be annoying but I like this style 🙌

Reply:
This is `black`, but black breaks lines if you terminate with a comma. If you delete the comma, black tidies it onto a single line. Don't worry, I'm letting Black do all the work for me...I'm not going to spend brainpower formatting my code!